parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -161,7 +161,7 @@ installStandaloneContainerd() {
         echo "currently installed containerd version ${CURRENT_VERSION} is greater than (or equal to) target base version ${desiredVersion}. skipping installStandaloneContainerd."
     else
         echo "installing containerd version ${desiredVersion}"
-        removeContainerd
+        removeContainerd || exit $ERR_CONTAINERD_INSTALL_TIMEOUT 
```

Comment:
Is it okay for us to exit here? Otherwise the error from `retrycmd_if_failure` in `removeContainerd` is ignored

Reply:
yes I think we should exit with error as you are doing here.

Reply:
cc @aadhar-agarwal / @anujmaheshwari1

Reply:
install_timeout error for `remove` command ?

Reply:
@djsly removed for now to align with how Ubuntu side handles this, we can adjust if needed later.